core/src/main/scala/dev/profunktor/fs2rabbit/algebra/Binding.scala

```diff
@@ -68,7 +68,7 @@ object Binding {
                                queueName: QueueName,
                                exchangeName: ExchangeName,
                                routingKey: RoutingKey): F[Unit] =
-        Sync[F].delay {
+        Sync[F].blocking {
```

Comment:
I think this could just stay the way it was as it wasn't using the `Blocker` previously
```suggestion
        Sync[F].delay {
```

Reply:
this is actually the place where you noticed that this delay is discarding the unbind #428 , so needs to be removed

Reply:
Yeah, I think in this case it prolly still makes sense to keep it as it was before, and let those changes be made separately.